src/policies/actions/PolicyActions.tsx

```diff
@@ -31,6 +34,16 @@ const PolicyActions: React.FC<PolicyActionsProps> = ({ policy, isKebabToggle })
   const [isEditModalOpen, setIsEditModalOpen] = React.useState(false);
   const [isDeleteModalOpen, setIsDeleteModalOpen] = React.useState(false);
 
+  const [canUpdatePolicy] = useAccessReview({
+    verb: 'update',
+    resource: NodeNetworkConfigurationPolicyModel.plural,
+  });
+
+  const [canDeletePolicy] = useAccessReview({
+    verb: 'update',
```

Comment:
Shouldn't be here 'delete' verb ? Otherwise it looks totally the same as the lines above..

Reply:
right  :D